node/pkg/accountant/accountant.go

```diff
@@ -283,15 +262,15 @@ func (acct *Accountant) publishTransferAlreadyLocked(pe *pendingEntry) {
 }
 
 // addPendingTransferAlreadyLocked adds a pending transfer to both the map and the database. It assumes the caller holds the lock.
-func (acct *Accountant) addPendingTransferAlreadyLocked(msgId string, msg *common.MessagePublication, digest string) error {
-	acct.logger.Debug("acct: addPendingTransferAlreadyLocked", zap.String("msgId", msgId))
-	if err := acct.db.AcctStorePendingTransfer(msg); err != nil {
+func (acct *Accountant) addPendingTransferAlreadyLocked(pe *pendingEntry) error {
+	acct.logger.Debug("acct: addPendingTransferAlreadyLocked", zap.String("msgId", pe.msgId))
+	pe.state.updTime = time.Now()
```

Comment:
I assume we should also acquire `pe.stateLock` here?  The caller only holds `acct.pendingTransfersLock` which doesn't protect against modification to the `pendingEntry` state.

Reply:
Fixed.